src/models/feedback.js

```diff
@@ -0,0 +1,40 @@
+const feedbackModel = (sequelize, DataTypes) => {
+  const Feedback = sequelize.define('Feedback', {
+    id: {
+      type: DataTypes.UUID,
+      defaultValue: sequelize.UUIDV4,
+      primaryKey: true
+    },
+    isLiked: {
+      type: DataTypes.BOOLEAN,
+      defaultValue: false,
+      validate: {
+        isBoolean: {
+          msg: 'like status must be a boolean value'
+        }
+      }
+    },
+    rating: {
+      type: DataTypes.INTEGER,
+      defaultValue: null,
+      validate: {
+        len: {
```

Comment:
I don't think `len` will work for this, `len` is to check the length of a value not the integer value. this will be better
```
max: {
          args: 5,
          msg: 'Ratings  must be less than 6'
        }
min: {
          args: 1,
          msg: 'Ratings  must be greater than 0'
      }
```

Reply:
Thanks for the feedback, It has been implemented.